zypp/misc/CheckAccessDeleted.cc

```diff
@@ -48,132 +49,181 @@ namespace zypp
 
     /** lsof output line + files extracted so far for this PID */
     typedef std::pair<std::string,std::unordered_set<std::string>> CacheEntry;
+  }
```

Comment:
I tried to keep the diff small on that file by not moving around the code more than required, so the the unnamed namespace is closed here and reopened later, also some impl members are not where they should be. I can also rearrange all the code if we do not care about diff size

Reply:
The diff size is not so important. But it's also not necessary to use a single unnamed namespace. I often use multiple consecutive unnamed namespaces to group different topics. Helper used for a single method are often located immediately before that method. 

What I like are comment lines around the namespace and repeating the namespace with the closing brace. (I know older code has even more code art, feel free to remove it)

    ///////////////////////////////////////////////////////////////////
    namespace detail
    {
    } // namespace detail
    ///////////////////////////////////////////////////////////////////

Reply:
While you're at it, please remove the dead code L252 ff. (lsofdebug).

Reply:
Done